hyperwallet/utils/apiclient.py

```diff
@@ -84,7 +94,7 @@ def _makeRequest(self,
             response = self.session.request(
                 method=method,
                 url=urljoin(self.baseUrl, url),
-                data=data,
+                data=(data if data is None else self.encryption.encrypt(data)) if self.encrypted else data,
```

Comment:
all this validation could be inside a method

Reply:
Moved to a method